testing/tests/DevExpress.ui.widgets.dataGrid/filtering.integration.tests.js

```diff
@@ -5,8 +5,19 @@ QUnit.testStart(function() {
             </div>
         </div>
     `;
+
+    const getEmulatorStyles = () => {
+        // Chrome DevTools device emulation
+        // Erase differences in user agent stylesheet
+        if('chrome' in window && devices.real().deviceType !== 'desktop') {
+            return 'input[type=date] { padding: 1px 0; }';
+        }
+
+        return '';
+    };
```

Comment:
Minor comment.

Maybe it will be better if we move this function to ```testing/helpers```.
(If this function will be used in other places of course)